troposphere/static/js/components/Header.react.js

```diff
@@ -187,12 +187,21 @@ let Header = React.createClass({
             });
             trackAction('switch-to-airport');
         };
+
+	var toggleBackground = {
+	color: "rgba(17, 5, 1, 0.3)",
+	width: 103,
+	height: 20,
+	borderRadius: 14,
+	fontSize: 13
+	};
+
         return (
           <div className="beta-toggle">
             <a href="/application?beta=false&airport_ui=true"
                 onClick={trackEvent}>
               <div className="toggle-wrapper">
-                <div className="toggle-background">
+                <div style={toggleBackground}>
```

Comment:
I believe Ish told me that moving from class to inline style was to ensure that the changes were _specific_ to just this component and no other.